front/pages/api/w/[wId]/keys/index.ts

```diff
@@ -18,6 +19,7 @@ export type PostKeysResponseBody = {
 
 const CreateKeyPostBodySchema = t.type({
   name: t.string,
+  group_id: t.union([t.string, t.undefined]),
```

Comment:
I'd prepare for later and support multiple group ids even though we only expose in the UI for now.

Reply:
this will require an association table to maintain the n-n relation ?
let's keep a single group to a key for now